.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -59,6 +59,6 @@ If the issue has an effect in the frontend, include any relevant screenshots and
 
 **Logs**
 
-Include relevant log snippets or files here. 
+Include relevant log snippets or files here.
 
-**I will abide by the [code of conduct] (https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md)**
+**I will abide by the [code of conduct] (https://github.com/fastruby/dash/blob/main/CODE_OF_CONDUCT.md)**
```

Comment:
I think we can do `**I will abide by the [code of conduct](./CODE_OF_CONDUCT.md)**` so it's easier to maintain. The link href works relative to the current file's directory https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-readmes#relative-links-and-image-paths-in-readme-files

Reply:
also, I think there's a space between `[code of conduct]` and the `(...)` breaking the link formatting (it was not introduced on this PR, it happens on main too, but maybe it's worth to fix it here)

Reply:
UPDATE on this: just tested and the way to use relative paths on a PR is `[code of conduct](../blob/main/CODE_OF_CONDUCT.md)`

I guess we still need the `main` part on the link